clients/pkg/promtail/targets/heroku/target.go

```diff
@@ -117,6 +117,12 @@ func (h *Target) drain(w http.ResponseWriter, r *http.Request) {
 			ts = message.Timestamp
 		}
 
+		// Create __param_<name> labels from query parameters
+		params := r.URL.Query()
+		for k, v := range params {
+			lb.Set(fmt.Sprintf("__param_%s", k), strings.Join(v, ","))
```

Comment:
Can you change the temporary label name to something like `__heroku_drain_param_%s` to be consistent with the other extracted labels.

Reply:
Sure